### PR TITLE
adjust background of night mode

### LINKED ISSUE #1886: Gray background for night reading mode إضافة الخلفية الرمادية لوضع القراءة الليلية
jazakum allahu khairan 
Please add the ability to customize the night reading mode and make the background of the Arabic reading in gray, as in this modified image, the gray color is more comfortable for the eye than the dark black.
جزاكم الله خيرا. رجاء إضافة إمكانية تخصيص وضع القراءة الليلية وجعل خلفية القراءة العربية باللون الرمادي كما بهذه الصورة المعدلة فاللون الرمادي مريح للعين افضل من الاسود المعتم.

### PR DESCRIPTION
**Pull request by benomaire**
should fix #1886 if you like the approach

### CODE REVIEW

**Review by ahmedre — APPROVED** [review]
this looks good to me - I'll need to make sure to apply it on the new page types also in sha' Allah, please remind me if I forget.

jazakumAllah khairan for doing this!
جزاكم الله خيراً!

File: app/src/main/res/xml/quran_preferences.xml [hunk 1]
```diff
@@ -66,6 +66,15 @@
         android:persistent="true"
         android:summary="@string/prefs_night_mode_text_brightness_summary"
         android:title="@string/prefs_night_mode_text_brightness_title"/>
+
+    <com.quran.labs.androidquran.ui.preference.SeekBarBackgroundBrightnessPreference
```
Comment (tahmid-23) [inline app/src/main/res/xml/quran_preferences.xml:70]: this class doesn't exist and causes error when settings is open

Comment (ahmedre) [reply app/src/main/res/xml/quran_preferences.xml:70]: #1916

Comment (benomaire) [reply app/src/main/res/xml/quran_preferences.xml:70]: @tahmid-23 Thank you for noting this. I forgot to stage the file!
@ahmedre Thank you for the fix and improvements!

**Comment by AlbahrawyTiger** [comment]
@benomaire @ahmedre @tahmid-23 
Alhamdulillah, thank you, my brothers.
jazakumAllah khairan
 I look forward to a new release eagerly to see this wonderful work.
I hope a new release will be before Ramadan.

**Pull request merged**